internal/spdxlicense/license.go

```diff
@@ -4,9 +4,26 @@ import (
 	"strings"
 )
 
+// https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-short-name
```

Comment:
thanks for adding the link; mind adding a little more verbiage around what this is and why there are these exceptions?